servers/zms/pom.xml

```diff
@@ -39,7 +39,7 @@
   </dependencyManagement>
 
   <properties>
-    <code.coverage.min>0.96</code.coverage.min>
+    <code.coverage.min>0.964</code.coverage.min>
```

Comment:
Increased precision so we'll be alerted immediately on new functionality not covered with tests